rust/noosphere/src/ffi/context.rs

```diff
@@ -104,6 +96,33 @@ pub fn ns_sphere_open(
     })
 }
 
+#[ffi_export]
+/// @memberof ns_sphere_t
+///
+/// Initialize an ns_sphere_t instance.
```

Comment:
What makes this opened as root? Separately, I'm not sure we use "root" anywhere else in public interface (not a for- or against-, but a note to myself to be consistent in terminology)

Reply:
This is just vestigial and needs to be removed, sorry!